src/TBuffer.cpp

```diff
@@ -3734,8 +3766,7 @@ QString TBuffer::bufferToHtml(QPoint P1, QPoint P2, bool allowedTimestamps, int
     if (allowedTimestamps && !timeBuffer[y].isEmpty()) {
         firstSpan = false;
         // formatting according to TTextEdit.cpp: if( i2 < timeOffset )
-        s.append(R"(<span style="color: rgb(200,150,0); background: rgb(22,22,22); )");
-        s.append(R"(font-weight: normal; font-style: normal; text-decoration: normal">)");
+        s.append("<span style=\"color: rgb(200,150,0); background: rgb(22,22,22); font-weight: normal; font-style: normal; text-decoration: normal\">");
```

Comment:
We're not using `tr()` here - why was the raw literal removed?

Reply:
I have a nasty suspicion that the raw literals are actually buggering up the parsing of the file for `tr(`...`)` instances **even when they are not involved in the using of those literals**.  As yet I have not found any third party utility that gets around this Qt Bug...